Test fail on windows (see appveyor) test_index_html

```
_______________________________ test_index_html _______________________________
tests\test_webui.py:28: in test_index_html
    assert resolve('/dir_with_index_html/') == (artifact, artifact_path)
E   AssertionError: assert ('dir_with_in...\\index.html') == ('dir_with_ind...l/index.html')
E     At index 1 diff: 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pytest-of-appveyor\\pytest-0\\test_index_html0\\webadmin\\dir_with_index_html\\index.html' != 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pytest-of-appveyor\\pytest-0\\test_index_html0\\webadmin\\dir_with_index_html/index.html'
E     Full diff:
E     ('dir_with_index_html/index.html',
E     -  'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pytest-of-appveyor\\pytest-0\\test_index_html0\\webadmin\\dir_with_index_html\\index.html')
E     ?                                                                                                                              ^^
E     +  'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pytest-of-appveyor\\pytest-0\\test_index_html0\\webadmin\\dir_with_index_html/index.html')
E     ?                             
```

--- a/lektor/admin/webui.py
+++ b/lektor/admin/webui.py
@@ -1,3 +1,5 @@
+import os
+import sys
 from flask import Flask, request, abort
 from flask.helpers import safe_join
 from werkzeug.utils import append_slash_redirect
@@ -65,7 +67,11 @@ class LektorInfo(object):
                 filename = artifact.dst_filename
 
         if filename is None:
-            filename = safe_join(self.output_path, path.strip('/'))
+            path_list = path.strip('/').split('/')
+            if sys.platform == 'win32':
+                filename = os.path.join(self.output_path, *path_list)
+            else:
+                filename = safe_join(self.output_path, *path_list)
 
         return artifact_name, filename
 

--- a/tests/test_webui.py
+++ b/tests/test_webui.py
@@ -23,8 +23,14 @@ def test_index_html(tmpdir, env):
     assert exc.value.response.status_code == 301
     assert exc.value.response.headers['Location'] == 'dir_with_index_html/'
 
-    artifact = 'dir_with_index_html/index.html'
-    artifact_path = os.path.join(info.output_path, artifact)
+    artifact_folder = 'dir_with_index_html'
+    artifact_item = 'index.html'
+    artifact = '{}/{}'.format(artifact_folder, artifact_item)
+    artifact_path = os.path.join(
+        info.output_path,
+        artifact_folder,
+        artifact_item,
+    )
     assert resolve('/dir_with_index_html/') == (artifact, artifact_path)
 
     # Same for index.htm as for index.html.
@@ -34,8 +40,14 @@ def test_index_html(tmpdir, env):
     assert exc.value.response.status_code == 301
     assert exc.value.response.headers['Location'] == 'dir_with_index_htm/'
 
-    artifact = 'dir_with_index_htm/index.htm'
-    artifact_path = os.path.join(info.output_path, artifact)
+    artifact_folder = 'dir_with_index_htm'
+    artifact_item = 'index.htm'
+    artifact = '{}/{}'.format(artifact_folder, artifact_item)
+    artifact_path = os.path.join(
+        info.output_path,
+        artifact_folder,
+        artifact_item,
+    )
     assert resolve('/dir_with_index_htm/') == (artifact, artifact_path)
 
     # Empty or absent directory has default resolution behavior.

